__tests__/integration/one-app.spec.js

```diff
@@ -862,6 +862,32 @@ describe('Tests that require Docker setup', () => {
       });
     });
 
+    describe('custom error page', () => {
+      const loadCustomErrorPageRoot = async () => {
+        await addModuleToModuleMap({
+          moduleName: 'frank-lloyd-root',
+          version: '0.0.4',
+        });
+        // wait for change to be picked up
+        await waitFor(5000);
+      };
+
+      afterAll(() => {
+        writeModuleMap(originalModuleMap);
+      });
+      describe('successful fetch of error page', () => {
```

Comment:
what about the default scenario ? when custom error is not provided ?

Reply:
I thought that was already tested here: https://github.com/americanexpress/one-app/blob/master/__tests__/integration/one-app.spec.js#L1263?